akka-actor-typed/src/main/scala/akka/actor/typed/internal/Supervision.scala

```diff
@@ -81,10 +72,15 @@ private abstract class AbstractSupervisor[I, Thr <: Throwable](strategy: Supervi
       val unwrapped = UnstashException.unwrap(t)
       strategy.logLevel match {
         case Logging.ErrorLevel =>
-          ctx.asScala.log.error(unwrapped, "Supervisor {} saw failure: {}", this, unwrapped.getMessage)
+          ctx.asScala.log.error("Supervisor {} saw failure: {}", Array(this, unwrapped.getMessage))
```

Comment:
For those we need to include the exception, using:
```
public void error(String msg, Throwable t)
```

which doesn't have support for variable substitutions, but since it's on error level it's fine to eagerly replace them with Scala string interpolation.

```
ctx.asScala.log.error(s"Supervisor $this saw failure: ${unwrapped.getMessage}", unwrapped)
```

Reply:
you're right

Reply:
I think I will go for @patriknw :
```ctx.asScala.log.error(s"Supervisor $this saw failure:", unwrapped)```